cvat/apps/dataset_manager/tests/test_formats.py

```diff
@@ -962,6 +962,6 @@ def test_can_import_mots_annotations_with_splited_masks(self):
             task.update()
             task = self._create_task(task, images)
 
-            dm.task.import_task_annotations(task['id'], dataset_path, format_name)
+            dm.task.import_task_annotations(task['id'], dataset_path, format_name, True)
```

Comment:
Do you plan to test the masks somehow?

Reply:
I, personally do not plan. I hope we will have some staff for that.